roles/openshift_hosted/tasks/router/router.yml

```diff
@@ -14,13 +14,22 @@
     openshift_hosted_router_selector: "{{ openshift.hosted.router.selector | default(None) }}"
     openshift_hosted_router_image: "{{ openshift.hosted.router.registryurl }}"
 
+# This is for when we desire a cluster signed cert
+# The certificate is generated and placed in master_config_dir/
+# We want this to run in an external include so that we can 
+# keep the logic in this playbook on how to handle certificates.
+- block:
+  - include: wildcard_router_cert.yml
+  when: openshift_hosted_router_create_certificate | default(false)
```

Comment:
Is the `default` needed here since it is in defaults/main.yml?  If so, then it seems it should also be at line 32.